apis/v1alpha1/backendpolicy_types.go

```diff
@@ -31,16 +31,25 @@ import (
 // implementation specific may be represented with similar implementation
 // specific custom resources.
 type BackendPolicy struct {
-	metav1.TypeMeta   `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
+	// +optional
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   BackendPolicySpec   `json:"spec,omitempty"`
+	// Spec defines the desired state of BackendPolicy.
+	//
+	// +optional
+	Spec BackendPolicySpec `json:"spec,omitempty"`
+
+	// Status defines the current state of BackendPolicy.
+	//
+	// +optional
+	// +kubebuilder:default={conditions: {{type: "Admitted", status: "False", message: "Waiting for controller", reason: "Waiting", lastTransitionTime: "1970-01-01T00:00:00Z"}}}
```

Comment:
I think we've already discussed this in another thread, but any form of condition on a BackendPolicy is going to be very hard to get right. This is a resource that may be used by zero-many controllers. I think this kind of default value makes sense for resources like Gateway that are clearly waiting for a single controller, but that's not the case here.

Reply:
ack